common/djangoapps/util/enterprise_helpers.py

```diff
@@ -120,3 +124,62 @@ def insert_enterprise_pipeline_elements(pipeline):
 
     for index, element in enumerate(additional_elements):
         pipeline.insert(insert_point + index, element)
+
+
+def get_enterprise_customer_logo_url(request):
+    """
+    Client API operation adapter/wrapper
+    """
+
+    if not enterprise_enabled():
+        return None
+
+    parameter = get_enterprise_branding_filter_param(request)
+    if not parameter:
+        return None
+
+    provider_id = parameter.get('provider_id', None)
+    ec_uuid = parameter.get('ec_uuid', None)
+
+    if provider_id:
+        branding_info = enterprise_api.get_enterprise_branding_info_by_provider_id(provider_id=provider_id)
```

Comment:
It looks like both `get_enterprise_branding_info_by_` methods can handle `NoneType` identifiers; if so, no reason to make these items conditional.